$NetBSD: patch-ay,v 1.3 2007/01/13 01:16:17 dmcmahill Exp $

This is needed for some non-gcc compilers such as the Sun 
Studio c-compiler

--- lib/ppm.h.orig	2006-04-29 18:13:01.000000000 -0400
+++ lib/ppm.h
@@ -233,7 +233,7 @@ struct hsv
 ppm_hsv_from_color(pixel  const color,
                    pixval const maxval);
 
-static __inline pixval
+static __inline__ pixval
 ppm_colorvalue(pixel const p) {
 /*----------------------------------------------------------------------------
   The color value (V is HSV) as a pixval
@@ -241,7 +241,7 @@ ppm_colorvalue(pixel const p) {
     return PPM_MAX(PPM_GETR(p), PPM_MAX(PPM_GETG(p), PPM_GETB(p)));
 }
 
-static __inline pixval
+static __inline__ pixval
 ppm_saturation(pixel const p,
                pixval const maxval) {
 /*----------------------------------------------------------------------------
